components/brave_wallet/browser/keyring_service.cc

```diff
@@ -784,8 +783,8 @@ void KeyringService::RestoreWallet(const std::string& mnemonic,
 
   if (IsFilecoinEnabled()) {
     // Restore mainnet filecoin acc
-    auto* filecoin_keyring = RestoreKeyring(mojom::kFilecoinKeyringId, mnemonic,
-                                            password, is_legacy_brave_wallet);
+    auto* filecoin_keyring =
+        RestoreKeyring(mojom::kFilecoinKeyringId, mnemonic, password, false);
```

Comment:
Here and below is_legacy_brave_wallet makes sense only when restoring eth keyring.